pkgs/borgbackup/borg/registry.yaml

```diff
@@ -23,7 +23,7 @@ packages:
         supported_envs:
           - linux/amd64
           - darwin
-      - version_constraint: semver("<= 1.1.18")
+      - version_constraint: semver("<= 1.2.8")
```

Comment:
Maybe rather use "< 1.4" (or so).

Some day there might be a 1.2.9 release, maybe even a 1.2.10.

Reply:
And those would use the old naming convention?

✅ changed

Reply:
Yes, within borg 1.2.x I will keep using the old naming convention.

borg 1.2.x will still be supported for a while, but likely I will stop making new 1.2.x releases once 1.4.x is practically proven to work well.

In general, borg 1.4 is quite similar to 1.2, kind of a "refreshed/modernised 1.2".